src/disasm.cpp

```diff
@@ -1050,10 +1054,14 @@ static void jl_dump_asm_internal(
             MCInst Inst;
             MCDisassembler::DecodeStatus S;
             FuncMCView view = memoryObject.slice(Index);
+#if JL_LLVM_VERSION >= 150000
```

Comment:
It's usually better to normalize to the newer version, so deleting the check ends up with the correct thing.